lib/core/wireprotocol/command.js

```diff
@@ -145,7 +145,7 @@ function supportsOpMsg(topologyOrServer) {
 }
 
 function _cryptCommand(server, ns, cmd, options, callback) {
-  const shouldBypassAutoEncryption = !!server.s.options.bypassAutoEncryption;
+  const shouldBypassAutoEncryption = !!server.s.bypassAutoEncryption;
```

Comment:
This is really just a pipedream, but my thought was that eventually we would be able to provide a `CryptoConnection` which subclasses the new `Connection` type - further allowing us to relegate everything about client encryption to the `mongodb-client-encryption` module. Is it possible to just look for `server.s.options.autoEncryption.bypassAutoEncryption` here? Or is that getting flattened? It would be awesome to keep encryption "stuff" relegated to as few places as possible in the driver code IMO

Reply:
This was originally:

```js
const shouldBypassAutoEncryption = !!(server.s.options.autoEncryption && server.s.options.autoEncryption.bypassAutoEncryption);
```

b/c we cannot be sure that server..autoEncryption will be there. We can put that in if you would prefer.

Reply:
Yeah, my thought was we could do that and then remove the code above that sets `this.s.bypassAutoEncryption` on the `Server` instance - now server has no concept of encryption